ruby/cross_service_examples/item-tracker/app.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require "sinatra"
+require "sinatra/cors"
+require_relative "db_wrapper"
+require_relative "report"
+require "aws-sdk-rdsdataservice"
+require "aws-sdk-ses"
+
+client = Aws::RDSDataService::Client.new
+ses_client = Aws::SES::Client.new
+config = YAML.load_file("helpers/config.yml")
+wrapper = DBWrapper.new(config, client)
+reporter = Report.new(wrapper, "fprior@amazon.com", ses_client)
```

Comment:
You probably don't want to publish your Amazon email address. 😁

Reply:
Yeah I don't know why I didn't put that on config to start with